src/main/groovy/io/seqera/wave/service/logs/BuildLogServiceImpl.groovy

```diff
@@ -97,6 +100,7 @@ class BuildLogServiceImpl implements BuildLogService {
     @Override
     void storeLog(String buildId, String content){
         try {
+            content = removeCondaLockFile(content)
```

Comment:
Thinking more about this, a better approach could be to extract the conda lock from the log and upload it in the same bucket just using a diffent key/extension just after the log. This would avoid having a "conda lock" service and extra logic to manage the surreal table. What do you think?

Reply:
this make sense, because conda lock depends upon log.

But it will make condalock coupled with log service, which make code less maintainable.
and if a customer wants to use different bucket for condalock, then this approach wont work.

we can have separate service for condalock and separate s3client and we can use the same bucket as logs in that client too. in this way it will be provide a way to use different bucket if required 
and also changes to log service wont affect condalock functionality

Reply:
Not that bad, the lesser option the better, and ultimately the lock cannot exist without the log file.

Reply:
understood, I will make the changes